sympy/physics/vector/tests/test_printing.py

```diff
@@ -41,37 +41,75 @@ def test_vector_pretty_print():
 
     expected = """\
  2
-a  n_x + b n_y + c*sin(alpha) n_z\
+a  n_x + b n_y + c*sin(alpha) n_z\n\
 """
     uexpected = u("""\
  2
-a  n_x + b n_y + c⋅sin(α) n_z\
+a  n_x + b n_y + c⋅sin(α) n_z\n\
 """)
-
-    assert ascii_vpretty(v) == expected
-    assert unicode_vpretty(v) == uexpected
-
-    expected = u('alpha n_x + sin(omega) n_y + alpha*beta n_z')
-    uexpected = u('α n_x + sin(ω) n_y + α⋅β n_z')
-
-    assert ascii_vpretty(w) == expected
-    assert unicode_vpretty(w) == uexpected
+    import sys
+    from sympy.core.compatibility import StringIO
+    result = StringIO()
+    old_stdout = sys.stdout
+    sys.stdout = result
+    try:
+        ascii_vpretty(v)
+    finally:
+        sys.stdout = old_stdout
```

Comment:
Don't capture stdout for the tests. If we don't have a separate function that just returns the values, at the very least, just test VectorPrettyPrinter().doprint.

Reply:
How can I test ```VectorPrettyPrinter().doprint``` with ```use_unicode``` set to False? This is required to assert with the strings 'expected'.